AmericanMedicalAssociation/ama-style-guide-2
# EWL-6207 Fix bullet position and style

## Ticket(s)
=
**Jira Ticket**
- [EWL-6207: Addition of SG2 styling of bullets per new zeplins](https://issues.ama-assn.org/browse/EWL-6207)

## Description
Match the bullets in list to the Zeplin doc
https://app.zeplin.io/project/59dd1f9c36a8b21d569bb9e8/screen/5bb67aa50a86693f117fc827

## To Test
- visit http://localhost:3000/?p=pages-news-with-jama-examples
- **observe the JAMA network blocks are the same** as https://americanmedicalassociation.github.io/ama-style-guide-2/?p=pages-news-with-jama-examples
- observe no other the social icon list looks the same
- We need to also test in D8
- to make sure you have the latest prod database run inside your vagrant `scripts/refresh-local -a one`
- set up your local.yml to use local SG2
- run `scripts/build`
- visit http://ama-one.local/delivering-care/diabetes/7-steps-get-prediabetic-patients-preventive-help-they-need
- observe the bullets are indented and match https://app.zeplin.io/project/59dd1f9c36a8b21d569bb9e8/screen/5bb67aa50a86693f117fc827
- Did you test in IE 11?

## Visual Regressions
A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/bugfix/EWL-6207-bullet-styling/html_report/index.html).



## Relevant Screenshots/GIFs
<img width="1032" alt="screen shot 2018-10-15 at 11 22 12 am" src="https://user-images.githubusercontent.com/2271747/46963950-90d93700-d06c-11e8-8cf3-673b088031f7.png">


## Remaining Tasks
Things relevant to here may be updating [The Glossary](https://issues.ama-assn.org:8446/confluence/display/DTD/Glossary?src=contextnavpagetreemode) with the latest naming changes or review of the relative [D8](https://github.com/AmericanMedicalAssociation/ama-d8) work.


## Additional Notes
n/a
---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
